test/spec/vanilla.spec.js

```diff
@@ -160,16 +172,17 @@ describe('backend with vanillajs', function() {
   it('should not serve up mock data when params do not match', function () {
     var xhr = new XMLHttpRequest();
 
-    backend.when('GET', 'fixtures/data.json', { yes: true }).respond({
+    backend.when('POST', 'api/theanswer', { yes: true }).respond({
       test: 'bye'
     });
 
-    xhr.open('GET', 'fixtures/data.json', false);
-    xhr.send({ yes: false });
+    xhr.open('POST', 'api/theanswer', false);
 
-    expect(JSON.parse(xhr.responseText)).to.eql({
-      test: 'hi'
-    });
+    try {
+      xhr.send({ yes: false });
```

Comment:
Also we should probably throw an error after this send to avoid a false positive